### ISSUE TITLE
SetBitmapAsync documentation should behavior when called in parallel

### ISSUE THREAD

**Issue by darthwing-duck** [MicrosoftDocs/winrt-api#2041]
co_await SetBitmapAsync(...);
co_await SetBitmapAsync(...); // cancels the above SetBitmapAsync

if you co_await 2x, will cancel the first IAsyncAction and this isn't mentioned in the documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4b31ac51-d271-e60a-01ee-9d3b3aba5ea7
* Version Independent ID: 3a73c16f-5b85-abdc-6e26-450de32e8fc8
* Content: [SoftwareBitmapSource.SetBitmapAsync(SoftwareBitmap) Method (Windows.UI.Xaml.Media.Imaging) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.imaging.softwarebitmapsource.setbitmapasync?view=winrt-20348#feedback)
* Content Source: [winrt-api-build/xml/Windows.UI.Xaml.Media.Imaging/SoftwareBitmapSource.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.UI.Xaml.Media.Imaging/SoftwareBitmapSource.xml)
* Product: **uwp**
* Technology: **winui**
* GitHub Login: @jwmsft
* Microsoft Alias: **jimwalk**

**Comment by GrantMeStrength** [comment]
Closing this issue due to age (filed before 2024). If this is still relevant, please reopen or file a new issue with updated details. Thank you for your contribution!
